lnrpc/routerrpc/router_server.go

```diff
@@ -375,6 +375,13 @@ func (s *Server) SendToRouteV2(ctx context.Context,
 		return rpcAttempt, nil
 	}
 
+	// Transform user errors to grpc code.
```

Comment:
Won't this possibly "break" behavior for those that rely on the current string-based error code?

Reply:
It changes the status code, but not the string message itself. If the caller doesn't rely on the grpc status code `Unknown` (which is changed to `AlreadyExists` in this pr) and only inspects the message, it should be good.